autoload/echodoc.vim

```diff
@@ -29,44 +29,56 @@ set cpo&vim
 " }}}
 
 " Variables  "{{{
+let s:debug = 1
```

Comment:
Left in by mistake.  I was using it to print the results of the new functions for testing.